ngraph_bridge/ngraph_builder.cc

```diff
@@ -3257,11 +3258,24 @@ Status Builder::TranslateGraph(
   //
   // Apply additional passes on the nGraph function here.
   //
-  ngraph::pass::Manager passes;
-  // passes.register_pass<ngraph::pass::ConstantFolding>();
-  passes.register_pass<pass::TransposeFolding>();
-  passes.register_pass<pass::TransposeSinking>();
-  passes.run_passes(ng_function);
+  {
+    ngraph::pass::Manager passes;
+    ngraph::pass::PassConfig pass_config;
+    // set/honor the defaults, unless specified via env var
+    auto set_default = [&pass_config](std::string pass, bool enable) {
+      auto enables_map = pass_config.get_enables();
+      if (enables_map.find(pass) == enables_map.end())
+        pass_config.set_pass_enable(pass, enable);
+    };
```

Comment:
Minor comment: This is fine, but if you change it to `get_enabled(std::string, bool default)` then there's only one change required to enable/disable a pass.